main/src/ca/uwaterloo/flix/api/Flix.scala

```diff
@@ -527,7 +528,7 @@ class Flix {
       (_, patMatchErrors) = PatMatch.run(afterStratifier)
       redundancyErrors = Redundancy.run(afterStratifier)
       (_, safetyErrors) = Safety.run(afterStratifier)
-      errors = instanceErrors ++ stratificationErrors ++ patMatchErrors ++ redundancyErrors ++ safetyErrors
+      errors: List[CompilationMessage & Recoverable] = regionErrors ::: entryPointErrors ::: instanceErrors ::: stratificationErrors ::: patMatchErrors ::: redundancyErrors ::: safetyErrors
```

Comment:
Perhaps this line is getting too long :)

Reply:
Yeah. I will fix it in another pr